devices/temp_nix.go

```diff
@@ -28,6 +33,38 @@ func getTemps(temps map[string]int) map[string]error {
 			temps[label] = int(sensor.Temperature)
 		}
 	}
+
+	block, err := ghw.Block()
+	if err != nil {
+		log.Print("error getting block device info")
```

Comment:
Wasn't sure what is the correct way to log errors

Reply:
Yes, sufficiently for a PR. I try to keep messages translat(able), but adding translated error messages is a non-trivial amount of extra effort, so it's fine.